### PR TITLE
#29073: Fix shellcheck issues in linux-tor-prio.sh

### PR DESCRIPTION
**Pull request by rl1987**
https://trac.torproject.org/projects/tor/ticket/29073#comment:1

### CODE REVIEW

File: contrib/operator-tools/linux-tor-prio.sh [hunk 1]
```diff
@@ -7,16 +7,16 @@
 ############################### README #################################
 
 # This script provides prioritization of Tor traffic below other
-# traffic on a Linux server. It has two modes of operation: UID based
```
Comment (tlyu) [inline contrib/operator-tools/linux-tor-prio.sh]: Please make the formatting changes in a separate commit.

Comment (rl1987) [reply contrib/operator-tools/linux-tor-prio.sh]: Fixed.

**Comment by tlyu** [comment]
Superseded by #804
